terraform/test-environments/scenarios/single_local_iamv2p1_inplace_upgrade.tf

```diff
@@ -0,0 +1,57 @@
+#
+# A simple local install of A2 with IAMv2 using the chef-automate CLI. It currently uses a pseudo-inplace upgrade.
+#
+
+module "single_local_iamv2p1_inplace_upgrade" {
+  source = "git@github.com:chef/es-terraform.git//modules/cd_instance_v2"
+
+  # DNS components ( a2-iamv2p1-local-inplace-upgrade-{{channel}}.cd.chef.co )
+  subdomain        = "a2-iamv2p1-local-inplace-upgrade"
+  subdomain_suffix = "-${var.dns_suffix}"
+
+  # Metadata
+  meta_title       = "Single Local (Inplace Upgrade) with IAMv2p1"
+  meta_description = "A2 stack with IAMv2p1 (using SAML) deployed locally as Habitat packages on a single host using the chef-automate CLI."
+  meta_type        = "habitat"
+
+  # AWS Instance Configuration
+  vpc            = "${var.environment}"
+  platform       = "ubuntu-16.04"
+  key_name       = "cd-infrastructure"
+  instance_type  = "m4.large"
```

Comment:
Is there a reason this is a m4.large while the other is a m5.large?

Reply:
Yes: because somebody started it that way. 😁

If you look at all the other "fresh-install / implace-upgrade" pairs, the fresh install uses **m5** and the inplace-upgrade uses **m4**. 

(I personally have no context what those values actually do or mean, though.)

Reply:
We should move to m5 large. We don't need to backfill now since that would recreate the instance, but going forward we should use m5 class instances.